exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -169,6 +169,10 @@ def __init__(
             insecure = False
 
         self._headers = headers or Configuration().EXPORTER_OTLP_HEADERS
+        if isinstance(self._headers, str):
```

Comment:
Any reason we are changing from `:` to `=` delimiters? No opinion, just curious. Also, once we have a place where all the env vars are documented, this specific format should probably be outlined.

Reply:
the delimiter as per the spec is `=`: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/exporter.md#specifying-headers-via-environment-variables